<?xml version="1.0"?>

<robot xmlns:xacro="http://www.ros.org/wiki/xacro">

  <transmission name="link_1_trans">
	<type>transmission_interface/SimpleTransmission</type>
	<joint name="joint1">
		<hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
	</joint>
	<actuator name="link_1_motor">
		<hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
		<mechanicalReduction>1</mechanicalReduction>
	</actuator>
  </transmission>

  <transmission name="link_2_trans">
	<type>transmission_interface/SimpleTransmission</type>
	<joint name="joint2">
		<hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
	</joint>
	<actuator name="link_2_motor">
		<hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
		<mechanicalReduction>1</mechanicalReduction>
	</actuator>
  </transmission>

  <transmission name="link_3_trans">
	<type>transmission_interface/SimpleTransmission</type>
	<joint name="joint3">
		<hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
	</joint>
	<actuator name="link_3_motor">
		<hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
		<mechanicalReduction>1</mechanicalReduction>
	</actuator>
  </transmission>

  <transmission name="link_4_trans">
	<type>transmission_interface/SimpleTransmission</type>
	<joint name="joint4">
		<hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
	</joint>
	<actuator name="link_4_motor">
		<hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
		<mechanicalReduction>1</mechanicalReduction>
	</actuator>
  </transmission>

</robot>
